app/Http/Controllers/Service/CentreController.php

```diff
@@ -34,4 +35,154 @@ public function printRegistrations(Centre $centre)
             ]
         );
     }
+
+    /**
+     * Exports a summary of registrations from the User's relevant Centres.
+     *
+     */
+    public function exportRegistrationsSummary()
+    {
+        // Get User
+        $user = Auth::user();
+
+        // Get now()
+        $now = Carbon::now();
+
+        // Get centres
+        $centres = $user->relevantCentres();
+
+        // get registrations and order them
+        $registrations = Registration::whereIn('centre_id', $centres->pluck('id')->all())
+            ->with(['centre','family.children','family.carers'])
+            ->orderBy('centre_id', 'asc')
+            ->get();
+
+        // set blank rows for laravel-excel
+        $rows = [];
+
+        // Looks like larevel-excel can't auto-generate headers
+        // by collating all the row keys and normalising.
+        // So we have to do it by hand.
+
+        // create base headers
+        $headers = [
+        ];
+
+        // Per registration...
+        foreach ($registrations as $reg) {
+            $row = [
+                "RVID" => $reg->family->rvid,
```

Comment:
might want to guard these by first making sure you geta  family back and a centre back etc. I have been bitten by this so many times

`"RVID" => $reg->family ? $reg->family->rvid : 'Family not found';`